app/src/main/java/com/team3663/scouting_app/utility/Logger.java

```diff
@@ -216,8 +216,15 @@ private void WriteOutEventFile(DocumentFile in_event_df) {
         // Form the output line that goes in the csv file.
         for (int i = 1; i < match_log_events.size(); ++i) {
             LoggerEventRow ler = match_log_events.get(i);
+
+            // Normalize the X, Y coordinates to be a %age of the image size
+            int normalized_x = 0;
+            int normalized_y = 0;
+            if (Constants.Match.IMAGE_WIDTH > 0) normalized_x = (int)(10_000.0 * Integer.parseInt(ler.X) / Constants.Match.IMAGE_WIDTH);
+            if (Constants.Match.IMAGE_HEIGHT > 0) normalized_y = (int)(10_000.0 * Integer.parseInt(ler.Y) / Constants.Match.IMAGE_HEIGHT);
```

Comment:
Where does the Hardcoded `10_000.0` come from?
Also you should not have Hardcoded numbers.

Reply:
Well, hard-coded values ARE allowed in calculations where it makes sense.  If you convert seconds to minutes, you can have a hard-coded value of 60 in there.  :)

10,000 is because we want to convert a percentage (0.3456) to a whole number with 2 digits of precision (3456).  I'll add a comment to explain that.

Reply:
OK thanks

Reply:
Below you directly changed out `ler.X` and `ler.Y` for your new `normalized_X` and `normalized_Y`, are they  roughly the same?
and is `Constants.Match.IMAGE_WIDTH` and `Constants.Match.IMAGE_HEIGHT` roughly equal 10,000?

Reply:
ler.x might be 124.   IMAGE_WIDTH is 1008.
normalized_x will be (124 / 1008) * 10,000 = 0.1230 * 10,000 = 1230

Reply:
does that help explain what's going on here?

Reply:
I'm just confused because previously in this situation would take in the `124` and now the same variable is using `1230`, that is a large difference.

Reply:
because it's normalized to a percentage now.  If you look at the bug, you'll see why we're doing this.  It allows up to modify the size of the image without messing up any previously collected data (it's still a percentage).  we can possibly use a phone, or other sized tablet with larger screen sizes, and it still all works with other collected data because it's all now a percentage, and not a pixel location (which can vary depending on the above situations).

Reply:
I understand why, I am just saying you replaced the 124 with a 1230 with out changing anything else.

Reply:
Is this fine because we are going to get rid of all the data collected before this, getting rid of the discrepancy?

Reply:
Yes, this is fine because this is for 2025 and beyond.  We saved off the app and reports for 2024, so we can use the "old stuff" if we need to.  However, to your point, we can't use old X,Y with new X,Y and I'm okay with that.  

We need to create new Tableau Flows for 2025 as well.  ("trap" no longer makes sense, for example - although we can use what we had as a starting point).

We haven't determined IF we want to use previous year data (and if so, WHAT data) so I'm fine crossing that bridge later.  I can't see us using previous year X and Y since the field layout is different year to year.